lnrpc/invoicesrpc/utils.go

```diff
@@ -160,6 +161,11 @@ func CreateRPCInvoice(invoice *invoices.Invoice,
 		rpcHtlcs = append(rpcHtlcs, &rpcHtlc)
 	}
 
+	// Perform an inplace sort of the HTLCs to ensure they are ordered.
+	sort.Slice(rpcHtlcs, func(i, j int) bool {
```

Comment:
We could use the new, generic compatible `slices` package:
```
	slices.SortFunc(rpcHtlcs, func(i, j *lnrpc.InvoiceHTLC) int {
		return cmp.Compare(i.HtlcIndex, j.HtlcIndex)
	})
```

Reply:
nice - done!